spec/base_decorater_spec.rb

```diff
@@ -0,0 +1,25 @@
+require_relative '../base_decorater'
+
+describe CapitalizeDecorator do
+  before(:each) do
+    @capitalize_decorator = CapitalizeDecorator.new('hello')
+  end
+
+  describe '#new' do
+    it 'returns a capitalize decorator object' do
+      expect(@capitalize_decorator).to be_an_instance_of(CapitalizeDecorator)
+    end
+  end
+end
+
+describe TrimmerDecorator do
+  before(:each) do
+    @trimmer_decorator = TrimmerDecorator.new('hello')
+  end
+
+  describe '#new' do
+    it 'returns a trimmer decorator object' do
+      expect(@trimmer_decorator).to be_an_instance_of(TrimmerDecorator)
+    end
+  end
+end
```

Comment:
- [ ]  Good job adding the tests team, but according to the previous reviewer comment you are missing a test for the `BaseDecorater` class. Please add one. 
- [Optional] Please notice you are testing the initializer method inside each class by saying `be_an_instance_of`, But we are expected to test the methods inside each class. like the `correct_name` method. 
- [Optional] Also It's expected to have a separate file for each class in our application. This provides clarity and visibility. Please keep that in mind.